.github/workflows/pr-verify.yaml

```diff
@@ -5,15 +5,64 @@ on:
     types: [opened, edited, synchronize, reopened]
 
 permissions:
-  checks: write # Allow access to checks to write check runs.
+  contents: read
+  pull-requests: write
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
-  verify:
+  verify-pr-type:
     runs-on: ubuntu-latest
-    name: verify PR contents
+    name: verify PR type
     steps:
-    - name: Verifier action
-      id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4 # tag=v0.4.3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # tag=v4.1.7
+
+      - name: Check if PR title has acceptable prefix
+        run:  |
+          targets=(":sparkles:" ":bug:" ":book:" ":seedling:" ":warning:" ":rocket:" ":running:" "✨" "🐛" "📚" "🌱" "⚠️" "🚀" "🏃")
```

Comment:
Sorry, slightly in contrast to what I wrote before. Can we move the implementation to a shell script in `hack/verify-pr-title.sh` please?
(the shell script should then just take the PR tilte as an argument)

This just makes it a lot easier to test the shell script manually a bit (and we get shellcheck linting for free)

Reply:
Very recently this scripped got added to kubebuilder: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/pr-title-checker.sh

Let's take a look if it makes sense to reuse it (please check if it covers all the checks we want)

Reply:
Thanks for good comment. The script that you mentioned only check if the title contains valid prefix.
So I also check the new script that checks the title doesn't contain the number. 👍